css/custom.css

```diff
@@ -2864,7 +2866,7 @@ footer a.white:hover, a.white:focus {
 }
 
 .footer-wrapper ul {
-    line-height: 1.7;
+    line-height: 3;
```

Comment:
This looks quite ugly. Maybe better change it only for mobile?

also:
![tordis](https://user-images.githubusercontent.com/28106476/119309880-3a462780-bc6f-11eb-81cd-72d2c742d845.png)

Reply:
ping @returndiego

Reply:
I fixed the issues you mentioned, @erciccione. I was unable to run the web server locally, so I made these changes sort of blindly. 

I will consider contributing more if the project is moved to a JavaScript/TypeScript platform such as [11ty](https://www.11ty.dev/). Ruby creates so much friction! Is there a good enough reason to use Ruby over Node.js when so much on the web uses JavaScript?

Reply:
> I was unable to run the web server locally

Did you take a loook at the instructions in the README?

> Is there a good enough reason to use Ruby over Node.js

The point is not the language used but the website generator. We used jekyll from the beginning, which is one of the most popular website generators

> when so much on the web uses JavaScript?

That's not necessarely a good thing. Getmonero doesn't use any javascript.